### PR TITLE
Add DCRDEX v0.6 press release

### PR DESCRIPTION
**Pull request by jzbz**

### CODE REVIEW

File: src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+title: "Decred DEX Launches DCRDEX 0.6, Offering new levels of Privacy and Security for Cross-Chain Swaps"
+date: 2023-04-18
+sortDate: 2023-04-18
+draft: false
+icon: "announcementsDefault.svg"
+---
+
+**[Chicago, IL, April 17, 2023]** Decred (DCR), a sovereignty and privacy maximizing currency, today released the latest version of its decentralized exchange. DCRDEX 0.6 offers direct peer-to-peer cross-chain swaps without any kind of intermediaries, solving key problems around privacy and security. The combination of privacy, resilience, and cross-chain compatibility in the new release is a first.
+
+New Assets
```
Comment (jholdstock) [inline src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md:11]:
```suggestion
#### New Assets

```

File: src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+The most notable new feature of DCRDEX 0.6 is the introduction of Ethereum and USDC. Users now have access to direct layer 1 atomic swaps with no intermediary, utility token, or third-party arbitration. When swapping from an asset like BTC, the funds are locked in a native contract of the user's creation that is never spendable by a third party, only the swap participants. As with all swaps performed by the software, no auxiliary chain or DEX-managed vault is used to trustlessly execute the swap.
+
+Full Self Custody
```
Comment (jholdstock) [inline src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md:14]:
```suggestion
#### Full Self Custody

```

File: src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+No Registration Fee
+Another significant change is the elimination of the one-time registration fee in favor of time-locked fidelity bonds. Instead of an initial sacrifice of funds for the ability to place orders, users now lock up funds on-chain for a certain amount of time after which the bond expires and the user redeems it.
+
+This change has a number of benefits, most notably that the bonds are only spendable by the user that posted them. This is essential as DCRDEX moves towards a fully-decentralized DEX mesh network with no rent-seeking mechanisms of any kind. The only cost of the swap is on-chain transaction fees. There is no intermediate utility token or liquidity pool, just another user and the assets' chains.
+
+“With centralized exchanges experiencing frequent hacks and outages, DCRDEX 0.6 is a timely and crucial development in the world of decentralized finance, said Brian Stafford, Blockchain Developer at Decred. “This latest release demonstrates that users can still exchange their funds without compromising the key principles of security, privacy, and self-custody.”
```
Comment (jholdstock) [inline src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md:27]:
```suggestion
> “With centralized exchanges experiencing frequent hacks and outages, DCRDEX 0.6 is a timely and crucial development in the world of decentralized finance, said Brian Stafford, Blockchain Developer at Decred. “This latest release demonstrates that users can still exchange their funds without compromising the key principles of security, privacy, and self-custody.”
```

File: src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md (hunk 1 above)
Comment (xaur) [reply src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md:11]: I think header's visual size (presentation) should be reduced by CSS and not by using a smaller *logical* level. Main title should be level 1 (`#`) and sub headings like New Assets should be level 2 (`##`).

**Review by jholdstock — APPROVED** [review]

File: src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md (hunk 1 above)
Comment (jholdstock) [reply src/content/en/news/2023-04-18_decred_releases_dcrdex_0.6.md]: Correct. I opened an issue for it.

**Comment by xaur** [comment]
Can we deploy this please?

**Pull request merged**